pages/linux/newgrp.md

```diff
@@ -0,0 +1,11 @@
+# newgrp
+
+> Switch primary group membership.
```

Comment:
Of who or what?

Reply:
@sbrl: is this a job for a fresh PR or can master still be restored?

Reply:
I think a fresh PR is needed here, yeah